client/src/components/TestSuite/TestSuiteDetails/TestGroupListItem.tsx

```diff
@@ -103,7 +107,7 @@ const TestGroupListItem: FC<TestGroupListItemProps> = ({
   );
 
   const expandedGroupListItem = (
-    <Accordion disableGutters className={styles.accordion}>
+    <Accordion disableGutters className={styles.accordion} expanded={(testGroup.result?.result == 'fail' || view == 'report') ? true : undefined}>
```

Comment:
I auto-expand fails in the normal view here (which doesn't catch all failey types yet).  I understand that this will 'lock' the accordion open, but when you run something and it fails it is very annoying to have to click through accordians to just get the error message.

Reply:
Note to make accordions controlled